### PR TITLE
✨Idealmedia new improvements

### PR DESCRIPTION
**Pull request by DenyHF**
Hello!
We implemented a couple of amp features into our code!
thanks!

### CODE REVIEW

**Comment by torch2424** [comment]
@DenyHF Will take a look on Monday 😄 Thank you very much for this contribution! 🎉

**Comment by torch2424** [comment]
Unfortunately, got pulled onto a super high priority project. 😢 Can you take this @zhouyx ?

**Comment by DenyHF** [comment]
Hi, thanks for the feedback.
Waiting for your decision.

File: 3p/integration.js [hunk 1]
```diff
@@ -273,6 +273,7 @@ const AMP_EMBED_ALLOWED = {
   dable: true,
   engageya: true,
   epeex: true,
+  idealmedia: true,
```
Comment (zhouyx) [inline 3p/integration.js]: Can I ask the reason to support `<amp-embed>`. This change migrate the `<amp-ad>` to using `<amp-embed>` instead. Thanks.

Comment (DenyHF) [reply 3p/integration.js]: we want to give our publishers the opportunity to integrate amp

Comment (zhouyx) [reply 3p/integration.js]: To help me understand they should already be able to integrate with `<amp-ad>`. What is the reason here to use `<amp-embed>`? Thanks.

Comment (DenyHF) [reply 3p/integration.js]: Thanks @zhouyx for your question 
   Our network has 2 different types of cooperation with partners 
1) Advertisement. We are showing ad using our widgets. 
For such cooperation we'd like to use <amp-ad>
2) News exchange. Such type of cooperation doesn't have any monetization so would like to use <amp-embed>

**Comment by zhouyx** [comment]
to @lannka on the amp-embed usage. What do you think?

**Comment by lannka** [comment]
`amp-embed` sounds good. I'm actually going to propose a removal of this whitelisting: #23172

**Comment by zhouyx** [comment]
Thank you @lannka for confirming. 
@DenyHF The change LGTM. You may need to fix the lint error before I merge the PR. 
You can try `gulp lint --local-changes --fix` Thanks

File: ads/_config.js [hunk 2]
```diff
@@ -485,7 +485,12 @@ export const adConfig = {
   'ibillboard': {},
 
   'idealmedia': {
-    renderStartImplemented: false,
+    renderStartImplemented: true,
```
Comment (zhouyx) [inline ads/_config.js]: @DenyHF Thank you for adding support to renderStart API. 
When I tested your example locally, I couldn't find the `render-start` msg to be sent. Could you please double check the implementation? Thank you.

Comment (DenyHF) [reply ads/_config.js]: @zhouyx Thanks!
This was a problem on our side. We have already fixed it.

Comment (zhouyx) [reply ads/_config.js]: Thank you @DenyHF 
I got this error when I test locally
image
Any ideas?

Comment (DenyHF) [reply ads/_config.js]: Thank!
Fixed!

**Comment by lannka** [comment]
@zhouyx is this ready for merge?

File: ads/_config.js (hunk 2 above)
Comment (zhouyx) [reply ads/_config.js]: Thank you @DenyHF, verified this is fixed.

**Pull request merged**